pkg/jira/jira.go

```diff
@@ -85,6 +89,26 @@ func (c *Verifier) ghUnlabeledPRs(extPR pr) ([]pr, error) {
 	return unlabeledPRs, nil
 }
 
+func (c *Verifier) commentOnPR(extPR pr, message string) (error, bool) {
+	// Get the comments from that PR
+	comments, err := c.ghIssueClient.ListIssueComments(extPR.org, extPR.repo, extPR.prNum)
+	if err != nil {
+		return err, false
+	}
+	// Check to see if the same message has already been posted.
+	for _, comment := range comments {
+		if strings.Contains(comment.Body, message) {
+			return nil, false
+		}
+	}
+	// If the message hasn't already been posted, post it.
+	err = c.ghCommentClient.CreateComment(extPR.org, extPR.repo, extPR.prNum, message)
```

Comment:
Use the updated `ghClient` instead (see the first comment).

Reply:
Fixed in https://github.com/openshift/release-controller/pull/515/commits/97e300f395f6bcf1c8e027c8b62ed7a632855ca8